lib/gruf/rspec/configuration.rb

```diff
@@ -72,11 +73,8 @@ def options
       # @return [Hash] options The reset options hash
       #
       def reset
-        VALID_CONFIG_KEYS.each do |k, v|
-          send("#{k}=", v)
-        end
-        self.rpc_spec_path = ::ENV.fetch('RPC_SPEC_PATH', DEFAULT_RSPEC_PATH).to_s
-        options
+        VALID_CONFIG_KEYS.each { |k, v| send("#{k}=", v) }
+        options.tap { |o| self.rpc_spec_path = o[:rpc_spec_path] }
```

Comment:
Hey @eiei0 - how are you loading your ENV variables into the application? `reset` here is called when the module is inherited (loaded), so as long as you are doing that before requiring the gem, you'll be fine.

There are two ways to do this if you're using file-based env like dotenv:
* use `require: false` in the gemspec and load `gruf/rspec` explicitly in the spec helper
* or use the `dotenv` binstub to ensure env is loaded before the main binstub runs

If you're just injecting ENV normally by prefixing: `env RPC_SPEC_PATH="/spec/gruf/" bundle exec rspec` this should work fine, as the ENV will be properly loaded then. Can you elaborate a bit on how you're loading gruf-rspec?

Reply:
Hey @splittingred. Thanks for the review!

At the top of our `spec/spec_helper.rb` file, we use `ENV["RPC_SPEC_PATH"] = "/spec/example/path"` to set the environment variable which does happen prior to requiring the gem in the same file. However, I don't believe that this is the main issue here. From what I can tell, the gem is getting the environment variable value just fine.
```
From: /Users/jonathanmacdonald/.asdf/installs/ruby/2.6.0/lib/ruby/gems/2.6.0/gems/gruf-rspec-0.2.2/lib/gruf/rspec/configuration.rb:34 :

    29:           base: Gruf::Rspec::AuthenticationHydrators::Base,
    30:           basic: Gruf::Rspec::AuthenticationHydrators::Basic
    31:         },
    32:         rpc_spec_path: ENV.fetch('RPC_SPEC_PATH', '/spec/rpc/').to_s,
    33:       }.freeze
 => 34:       require 'pry'; binding.pry
    35:
    36:       attr_accessor *VALID_CONFIG_KEYS.keys
    37:
    38:       ##
    39:       # Whenever this is extended into a class, setup the defaults

[1] pry(Gruf::Rspec::Configuration)> ENV.fetch("RPC_SPEC_PATH")
=> "/spec/example/path"
```

As @almogriskified mentioned in #7, the problem is that when the module is included, `reset` is called and sets `rpc_spec_path` to a hardcoded string instead of whatever is configured via the environment variable.

Unless we're doing something wrong, I think the environment variable is basically ignored when loading the configuration. My apologies for not making that clear in the summary.

Reply:
Oh, wait! I'm realizing that you may have already fixed this in d3a48fd ([here](https://github.com/bigcommerce/gruf-rspec/commit/d3a48fd844e8b03beb50e4a6822582132701fe1b#diff-34f824f1682a1387a7d360d6f9182c9503178193118aa6b149fc1b4fda7bf107R78)).

I think our fork was out of date with the hardcoded string still present in `reset`.

Reply:
By changing our `Gemfile` to look something like the following, everything works as expected:
```
group :test do
  ...
  gem "gruf-rspec", git: "https://github.com/bigcommerce/gruf-rspec", branch: "main"
end
```

So I think _maybe_ the only thing we need in order for this issue to be resolved is a version bump? Thoughts @splittingred?

Reply:
Yep, I think that'll do. I'll cut a 0.2.3 now. :+1: Thanks for digging in!